docs/source/en/api/pipelines/audioldm.mdx

```diff
@@ -12,73 +12,40 @@ specific language governing permissions and limitations under the License.
-import torch
-import scipy
+The original codebase can be found at [haoheliu/AudioLDM](https://github.com/haoheliu/AudioLDM), and the pipeline was contributed by [sanchit-gandhi](https://huggingface.co/sanchit-gandhi). 
 
-repo_id = "cvssp/audioldm-s-full-v2"
-pipe = AudioLDMPipeline.from_pretrained(repo_id, torch_dtype=torch.float16)
-pipe = pipe.to("cuda")
+## Tips
 
-prompt = "Techno music with a strong, upbeat tempo and high melodic riffs"
-audio = pipe(prompt, num_inference_steps=10, audio_length_in_s=5.0).audios[0]
+When constructing a prompt, keep in mind:
 
-# save the audio sample as a .wav file
-scipy.io.wavfile.write("techno.wav", rate=16000, data=audio)
-```
+* Descriptive prompt inputs work best; you can use adjectives to describe the sound (for example, "high quality" or "clear") and make the prompt context specific (for example, "water stream in a forest" instead of "stream").
+* It's best to use general terms like "cat" or "dog" instead of specific names or abstract objects the model may not be familiar with.
 
-### Tips
+During inference:
 
-Prompts:
-* Descriptive prompt inputs work best: you can use adjectives to describe the sound (e.g. "high quality" or "clear") and make the prompt context specific (e.g., "water stream in a forest" instead of "stream").
-* It's best to use general terms like 'cat' or 'dog' instead of specific names or abstract objects that the model may not be familiar with.
-
-Inference:
-* The _quality_ of the predicted audio sample can be controlled by the `num_inference_steps` argument: higher steps give higher quality audio at the expense of slower inference.
+* The _quality_ of the predicted audio sample can be controlled by the `num_inference_steps` argument; higher steps give higher quality audio at the expense of slower inference.
 * The _length_ of the predicted audio sample can be controlled by varying the `audio_length_in_s` argument.
 
-### How to load and use different schedulers
-
-The AudioLDM pipeline uses [`DDIMScheduler`] scheduler by default. But `diffusers` provides many other schedulers
-that can be used with the AudioLDM pipeline such as [`PNDMScheduler`], [`LMSDiscreteScheduler`], [`EulerDiscreteScheduler`],
-[`EulerAncestralDiscreteScheduler`] etc. We recommend using the [`DPMSolverMultistepScheduler`] as it's currently the fastest
-scheduler there is.
+<Tip>
 
-To use a different scheduler, you can either change it via the [`ConfigMixin.from_config`]
-method, or pass the `scheduler` argument to the `from_pretrained` method of the pipeline. For example, to use the
-[`DPMSolverMultistepScheduler`], you can do the following:
+Make sure to check out the Schedulers [guide](/using-diffusers/schedulers) to learn how to explore the tradeoff between scheduler speed and quality, and see the [reuse components across pipelines](/using-diffusers/loading#reuse-components-across-pipelines) section to learn how to efficiently load the same components into multiple pipelines.
 
-```python
```

Comment:
Where can the user find this kind of example code now?

Reply:
This would be shown in the pipeline's [`__call__`](https://moon-ci-docs.huggingface.co/docs/diffusers/pr_3905/en/api/pipelines/audioldm#diffusers.AudioLDMPipeline.__call__.example) method kind of like how the example usage is shown in the `forward` for a model in Transformers.

Reply:
Is this the case for all pipelines? If yes, I think we can indeed remove it

Reply:
Yup! For example, [Text-to-image model editing](https://moon-ci-docs.huggingface.co/docs/diffusers/pr_3905/en/api/pipelines/model_editing), [Semantic guidance](https://moon-ci-docs.huggingface.co/docs/diffusers/pr_3905/en/api/pipelines/semantic_stable_diffusion), [RePaint](https://moon-ci-docs.huggingface.co/docs/diffusers/pr_3905/en/api/pipelines/repaint), etc.

Reply:
ok for me!